### DISCUSSION TITLE
WalletConnect V2 Flutter documentation not available

### DISCUSSION THREAD

**Discussion by Aastha9018** [WalletConnect/walletconnect-monorepo#3392] [Developer Support]
Hey, I have tried really hard but there's no proper documentation for the flutter app integration with wallet Connect v2. Can you please help me with some of the links.
Also the documentation present on the official website is just screenshots of snippets which aren't really healpful and then when i try tp google some bugs that I'm facing, there's no material for that as well. Please help me.

LAstly can you help me to any documentation maybe flutter/react-native or any other where the event handling is metioned for `wallet_addEthereumChain`.

**Comment by glitch-txs** [comment]
hi, where you able to check [this section](https://docs.walletconnect.com/2.0/flutter/walletconnectmodal/about) in the docs? Also is there any specific issue you're facing?

**Comment by Aastha9018** [comment]
Hi,

I am using the example app in the [Flutter](https://pub.dev/packages/walletconnect_flutter_v2).
I am running the wallet app in my local system.
I tried connecting it to Remix. The default chain that it gets connected to is Arbitrum(42161). After trying to switch it to Georli, now I'm trying to deploy a contract in Remix but it gives me the error. The first time it says that Invalid/Missing parameters and the second time that the `eth_sendTransaction` method doesn't exist. You can see the screenshots.

In the Web3Client, I have changed the URL from `https://mainnet.infura.io/v3/51716d2096df4e73bec298680a51f0c5` to `https://goerli.infura.io/v3/`, still it fails.
I googled the error, and it states that using Infura RPC URLs maybe give the error, so I changed it to `https://rpc-mumbai.maticvigil.com/v1/83ca0e94a4324ee57e324ceb6cb1a41b2420cbda` still the same issue. 
Whereas if I try connecting to the WalletConnect official app, it succeeds: https://react-app.walletconnect.com/

Can you please help me resolve it?
Much appreciated, thanks.

<img width="1355" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/22736786/4ca28e44-2486-49f0-bd32-21e6363b457c">
<img width="1016" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/22736786/d4cdded2-1e95-4ec2-b307-96bc010ecbfa">
